docs/_data/components/schemas/server-generated-id.yaml

```diff
@@ -0,0 +1,10 @@
+type: string
+description: |
+
+  Unique identifier for the object, as generated by the TROLIE server, typically as a result of POST
+  requests.  For POST requests, this should not be provided, as it will be generated by the server and
```

Comment:
An implication here is that the identifiers *should* be provided in `PUT` requests, strictly speaking. However, the examples for `PUT`s reuse the `POST` examples. I believe that `PUT` should be allowed to have the `id` property, so I suggest we require it in the spec and let implementations decide if they want to be lax in allowing `PUT` without an `id`.

Reply:
I might be confused- if you require it in the spec, that means it has to be provided on `POST` too, right?  Or we'd have to have a parallel schema structure?  Or are you suggesting I just change the examples?

Reply:
I just changed the PUT to use the GET example. I think that works.
https://github.com/getorymckeag/spec/pull/1/files#diff-ff3c49b7899f9c2126a8ac2eb746e7ded3573f4bd8f66ca6d87995524e12f31cR81

Reply:
but yes, that also meant adding a "request" for each temp rating which was pretty clean after the yaml refactoring
https://github.com/getorymckeag/spec/pull/1/files#diff-5f94b82995017623bf9b6670094a3398d175220f0f41a218d130f9d928c5f9f5R49

Reply:
Yep.  I like it.